device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/https/HttpsConnectionTest.java

```diff
@@ -394,27 +394,21 @@ public void setSSLContextSetsContextWithProxy(@Mocked final SSLContext mockedCon
     }
 
     //Tests_SRS_HTTPSCONNECTION_34_026: [If this object uses HTTP, this function shall throw an UnsupportedOperationException.]
-    @Test (expected = UnsupportedOperationException.class)
+    @Test (expected = IllegalArgumentException.class)
     public void setSSLContextThrowsIfHttp(@Mocked final SSLContext mockedContext) throws IOException, TransportException
```

Comment:
Since this test method now checks that the constructor throws, you can rename this test method

Reply:
I totally spaced out on the name ! sure thing